scripts/split-e2e-tests.ts

```diff
@@ -245,7 +255,8 @@ const splitTests = (baseJobLinux: any, testDirectory: string, pickTests?: (testS
 
       if (RUN_SOLO.find((solo) => test === solo || test.match(solo))) {
         if (RUN_IN_ALL_REGIONS.find((allRegionsTest) => test === allRegionsTest || test.match(allRegionsTest))) {
-          const candidateRegions = filterCandidateRegions(test, testRegions);
+          // always run these jobs in regions that do not have the beta layer deployed
+          const candidateRegions = filterCandidateRegions(test, testRegions, false);
```

Comment:
The canaries run solo across regions and the SQL ones might fail if run in non-supported beta regions

Reply:
The canaries do not use the lambda layer so they will pass fine in this region. We can see successfull runs here: 
* https://us-east-1.console.aws.amazon.com/codesuite/codebuild/594813022831/projects/amplify-category-api-e2e-workflow/build/amplify-category-api-e2e-workflow%3A3c17047b-e41c-433f-9d6f-86be971fdd9a?region=us-east-1
* https://us-east-1.console.aws.amazon.com/codesuite/codebuild/594813022831/projects/amplify-category-api-e2e-workflow/build/amplify-category-api-e2e-workflow%3Aff30ddf6-99cd-4b84-b1b2-bf0926e207a4?region=us-east-1